### DISCUSSION TITLE
How to compile on Visual Studio v110?

### DISCUSSION THREAD

**Discussion by aurixinino** [serge1/ELFIO#69] [Q&A] [answered]
I have Microsoft Visual Studio 2012 Professional, but I see that Elfio requires v142. Is there any possibility to still use MS Visual Studio v110?

**Comment by serge1** [comment, marked as answer]
ELFIO library has no specific requirements for a compiler version. Just make sure that the compiler supports C++11 language standard.

In addition, Visual Studio project files are not a part of the library distribution now. Please use the provided CMake files to generate corresponding project.
